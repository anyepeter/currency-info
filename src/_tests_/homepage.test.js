import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import Home from '../components/homepage';
import store from '../redux/configureStore';

describe('homepage ', () => {
  it('check if it rendering or not', () => {
    const { container } = render(
      <Provider store={store}>
        <Home />
      </Provider>,
    );
    expect(container).toMatchSnapshot();
  });
});
