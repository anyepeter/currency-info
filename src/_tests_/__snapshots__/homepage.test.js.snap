// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`home check if it rendering or not 1`] = `
<div>
  <div
    class="container"
  >
    <input
      class="input"
      placeholder="search a coin"
      type="text"
      value=""
    />
  </div>
  <div
    class="grid"
  />
</div>
`;

exports[`n check if it rendering or not 1`] = `
<div>
  <div
    class="container"
  >
    <input
      class="input"
      placeholder="search a coin"
      type="text"
      value=""
    />
  </div>
  <div
    class="grid"
  />
</div>
`;

exports[`navbar render links check if it rendering or not 1`] = `
<div>
  <div
    class="container"
  >
    <input
      class="input"
      placeholder="search a coin"
      type="text"
      value=""
    />
  </div>
  <div
    class="grid"
  />
</div>
`;
